lib/xypn_seo/xypn_scraper/scrape_advisor_profile_page.rb

```diff
@@ -19,29 +18,27 @@ def self.parse(html_string, xypn_url)
 
       attr_reader :nokogiri_object
 
-      def self.create_nokogiri_object(html_string)
+      def create_nokogiri_object(html_string)
         clean_html = HTMLCleaner.clean(html_string)
         @nokogiri_object = Nokogiri::HTML(clean_html)
       end
 
-      def self.parse_advisor_name
+      def parse_advisor_name
         @nokogiri_object.xpath('//h1').children.first.text.strip
       end
 
-      def self.parse_business_name
+      def parse_business_name
         @nokogiri_object.xpath('//h1').children.last.text
       end
 
-      def self.parse_business_site
-        begin
-          @nokogiri_object.xpath('//p[@class="advisor-website"]/a/@href').first.value
-        rescue Exception => e
-          Rails.logger.error(e)
-          return "Business URL Not Found" 
-          # db placeholder, clean view rendering. Evaluation handles the API error.
-        end
+      def parse_business_site
+        @nokogiri_object.xpath('//p[@class="advisor-website"]'\
+          '/a/@href').first.value
+      rescue StandardError => e
+        Rails.logger.error(e)
+        return 'Business URL Not Found'
```

Comment:
I checked in the DB, and as it stands the current function is leaving the space 'nil', so we're good there.